core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -56,11 +60,43 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
             REVIEW_TEST_DATA_URL, {
               story_id: ctrl.storyId
             });
+          var reviewTestsUrl = UrlInterpolationService.interpolateUrl(
+            REVIEW_TESTS_URL, {
+              story_id: ctrl.storyId
+            });
+          var storyViewerUrl = UrlInterpolationService.interpolateUrl(
+            STORY_VIEWER_PAGE, {
+              story_id: ctrl.storyId
+            });
           $http.get(reviewTestsDataUrl).then(function(result) {
+            var skillIdList = [];
+            var skillDescriptions = [];
+            for (var skillId in result.data.skill_descriptions) {
+              skillIdList.push(skillId);
+              skillDescriptions.push(
+                result.data.skill_descriptions[skillId]);
+            }
             var questionPlayerConfig = {
-              skillList: result.data.skill_ids,
+              resultActionButtons: [
+                {
+                  type: 'BOOST_SCORE',
+                  text: 'Boost My Score'
+                },
+                {
+                  type: 'RETRY_SESSION',
+                  text: 'New Test',
+                  url: reviewTestsUrl
+                },
+                {
+                  type: 'DASHBOARD',
+                  text: 'Return To Story',
+                  url: storyViewerUrl
+                }
+              ],
+              skillList: skillIdList,
```

Comment:
Add questionPlayerMode to this config. (See above comment of the values for questionPlayerMode)

Reply:
Done.